advanced/creating-a-skin.md

```diff
@@ -7,78 +7,80 @@ description: Introducing skin creation, less and icon modification.
 
-> Important: TinyMCE does not recommend modifying or overriding CSS rules directly.
+> Important: We do not recommend modifying or overriding CSS rules directly.
 
-1. Download (or `git clone`) the [TinyMCE skin tools](https://github.com/tinymce/oxide).
-1. Open a terminal and `cd` into the `/oxide` directory.
-1. Run the following command to install all dependencies:
-```js
-npm install
-```
-4. To start `gulp` and spin up a web server for previewing the files, run the following command:
-```js
-npm run start
-```
-> This command starts the preview server at the URL `http://localhost:3000` and starts watching the filesystem for changes.
+To set up the skin development environment, begin with the following steps:
 
-> Note: Run the following command to just build the files:
-```
-npm run build
-```
+1. Download (or `git clone`) the [TinyMCE source code](https://github.com/tinymce/tinymce).
 
+2. Open the terminal and navigate to the folder you just downloaded
 
-### Result:
+3. Install dependencies with the command:
+
+   ```
+    yarn install
+   ```
+
+4. Launch the web server to preview the skins using the command:
+
+   ```
+   yarn oxide-start
+   ```
+
+You should now be able to open a web browser and point it to the url displayed in the terminal, usually `http://localhost:3000`.
 
 The development environment is set up and ready to work.
 
 ![**TinyMCE skin SDK for Silver theme**]({{site.baseurl}}/images/SDKforsilver.png)
 
+If you just need to build the skins without launching a web server, run:
+```
+yarn oxide-build
+```
+
 ## Making or editing a skin
 
-The preparation steps above must be performed to create or edit a skin.
+Make sure you have performed the preparation steps above.
 
 ### Overview
 
-In the `/oxide` directory, navigate to `src/less/skins/`. There are two folders in this location:
-* `/ui` - which is the skins for the editor.
+Navigate to `modules/oxide/src/less/skins/`. There are two folders in this location:
+* `/ui` - which is the skins for the editor. The important file here is  `skin.less`.
 * `/content` - which is the skins for the content within the editor.
 
-The skin imports the theme `less` files located in the `src/less/theme/` directory.
+The folder `modules/oxide/src/less/theme/` contains the less files. At the top of most files you find the available variables that defines the default colors, margins, fonts etc (variables are the strings that starts with an at-character, for example `@background-color`). *Do not edit these files*, instead use them as a reference when creating your skin. Two common files you likely will start with are the global variables: `modules/oxide/src/less/theme/globals/global-variables.less` and the toolbar buttons: `modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less`.
```

Comment:
`At the top of most files you find the available variables` -> `At the top of most files you'll find the available variables` maybe?

Reply:
`less` should probably also be capitalised, in the first sentence of this paragraph. I read it as the "less than" definition at first, and got a bit confused. 

`Two common files you likely will start with are the global variables:` is a bit awkward to read too. Might be better as something like `We recommend starting with the two files containing global variables:`?